README.md

```diff
@@ -8,12 +8,52 @@ This repo contains a set of common bash scripts to be used in the delivery pipel
 ```sh
 git clone git@github.com:Financial-Times/next-ci-shared-helpers.git
 cd next-ci-shared-helpers
-make install
 ```
 
 
 ## Development
 
+### Working locally
+
+Many of the scripts require a Unix environment so in order to run existing scripts you'll need to be familiar with [Docker](https://www.docker.com/get-started).
+
+The first step is to [install docker](https://www.docker.com/get-started) and once you're up and running you'll be able to run the scripts locally inside a Docker container.
+
+For example, the following command will run the `helper-install-awscli` script inside the container.
+
+```shell
+docker run --rm -it -w /tmp --mount type=bind,source="$(pwd)",destination=/tmp circleci/node:8-browsers ./helper-install-awscli
```

Comment:
What is the expected output? I am trying on my machine and getting a response that does not look like that is intended.

Also might be worthwhile using a more recent Node version as an example to keep our documentation up-to-date.

Reply:
@andygout Sure i'll update it to LTS

The output for the script in the example is the setup/install logs for `awscli`. The last line you should see is:
```
Setting up awscli (1.11.13-1) ...
```

Do you see something else?